cmd/buildah/bud.go

```diff
@@ -115,14 +115,10 @@ func budCmd(c *cli.Context) error {
 	if len(dockerfiles) == 0 {
 		dockerfiles = append(dockerfiles, filepath.Join(contextDir, "Dockerfile"))
 	}
-	if err := parse.ValidateFlags(c, buildahcli.BudFlags); err != nil {
-		return err
-	}
-	if err := parse.ValidateFlags(c, buildahcli.LayerFlags); err != nil {
-		return err
-	}
-	if err := parse.ValidateFlags(c, buildahcli.FromAndBudFlags); err != nil {
-		return err
+	for _, flag := range [][]cli.Flag{buildahcli.BudFlags, buildahcli.LayerFlags, buildahcli.FromAndBudFlags} {
+		if err := parse.ValidateFlags(c, flag); err != nil {
+			return nil
```

Comment:
Shouldn't we still be returning err here and not nil?  Beyond which even though this is more efficient, I think having them broken out like they were makes it easier to find where the issue was, but could be proven wrong on that.

Reply:
Hi @TomSweeneyRedHat 

Thanks for noticing about the `nil` thing. I put it there to check something and forgot to change it back.
I am not sure what you meant in the last sentence, can you be more specific?

Thanks again :+1: